src/components/media/MediasSelectModal.jsx

```diff
@@ -54,8 +73,38 @@ const titleCase = (str) => {
   return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase()
 }
 
+const FolderBreadcrumb = ({ mediaDirectoryName, setQueryParams }) => (
+  <VStack>
+    <Breadcrumb alignSelf="start">
+      {mediaDirectoryName &&
+        mediaDirectoryName.split("%2F").map((dir, idx) => (
+          <BreadcrumbItem
+            isCurrentPage={idx === mediaDirectoryName.split("%2F").length - 1}
+          >
+            <BreadcrumbLink
+              onClick={(e) => {
+                e.preventDefault()
+                setQueryParams((prevState) => ({
+                  ...prevState,
+                  mediaDirectoryName: getMediaDirectoryName(
+                    mediaDirectoryName,
+                    { end: idx + 1 }
+                  ),
+                }))
+              }}
+            >
+              {deslugifyDirectory(dir)}
+            </BreadcrumbLink>
+          </BreadcrumbItem>
+        ))}
+    </Breadcrumb>
+  </VStack>
+)
+
 const MediasSelectModal = ({
   onProceed,
+  onExternalProceed,
```

Comment:
i think this parameter might be safe to remove with the new changes

Reply:
This is needed! The select modal has differing behaviour for ext and internal images - ext puts it in the file immediately, internal brings up the alt text modal